stake-pool/program/src/processor.rs

```diff
@@ -410,6 +410,18 @@ impl Processor {
             stake_program_info.clone(),
         )?;
 
+        Self::stake_authorize(
```

Comment:
nice catch -- did you notice this from testing?

Reply:
Yes and then verified with the stake program source code. We need Staker permission for a lot of things...